Source/Cmlx/mlx-generated/metal_compiled_preamble.cpp

```diff
@@ -412,6 +412,12 @@ constexpr complex64_t operator/(complex64_t a, complex64_t b) {
 constexpr complex64_t operator%(complex64_t a, complex64_t b) {
```

Comment:
This file and the previous are generated from mlx -- needed to support lack of build-time generation for non-swift files.